package-lock.json

```diff
@@ -5100,6 +5100,239 @@
 				"react-dom": "^16.9.0"
 			}
 		},
+		"@wordpress/env": {
+			"version": "file:packages/env",
+			"dev": true,
+			"requires": {
+				"chalk": "^2.4.2",
+				"docker-compose": "^0.22.2",
+				"nodegit": "^0.26.2",
+				"ora": "^4.0.1",
+				"terminal-link": "^2.0.0",
+				"yargs": "14.0.0"
+			},
+			"dependencies": {
```

Comment:
Those dependencies aren't hoisted to the root `node_modules` folder. I guess some of the dependencies use a different version in other packages or in the root `packages.json` file. Can we align them?

Reply:
https://github.com/WordPress/gutenberg/pull/17705 addresses it.

Reply:
Nice